minique/cli.py

```diff
@@ -16,7 +17,7 @@ def get_parser():
     return parser
 
 
-def main(argv=None):
+def main(argv: Optional[List[str]] = None) -> None:
```

Comment:
Why have `None` as an explicit default for an optional param? (Or: why was this default here in the first place?)

Reply:
Um, what? How would you otherwise define the optional parameter?

Reply:
Nevermind, I misthought! 😅